src/servers/auth-router.ts

```diff
@@ -16,14 +16,15 @@ const graphqlLogger = baseLogger.child({
   module: "graphql",
 })
 
-const { serverURL, corsAllowedOrigins } = getKratosConfig()
-
 const authRouter = express.Router({ caseSensitive: true })
 
+// TODO: why is cors origin policies mapped to kratos config?
```

Comment:
Because its cors for the endpoint that exchanges the cookie from the web wallet to a backend signed jwt. Its literally cors only for the kratos related endpoint called from the web-wallet.